docs/contributing/main.rst

```diff
@@ -0,0 +1,46 @@
+.. _contributing:
```

Comment:
You made an excellent reordering of the developer documentation but as I mentioned in the bug report we really need to have fewer files, all the "contributing" section of the documentation should be in one file. Otherwise we quickly end up with outdated documentation.

Reply:
Ok, most pages are concatenates.